aptos-move/aptos-vm/src/storage_adapter/executor_view_base.rs

```diff
@@ -9,25 +9,36 @@ use aptos_state_view::{StateView, StateViewId};
 use aptos_types::state_store::{
     state_key::StateKey, state_storage_usage::StateStorageUsage, state_value::StateValue,
 };
-use aptos_vm_types::resolver::{StateStorageView, TModuleView, TResourceView};
-use move_core_types::value::MoveTypeLayout;
+use aptos_vm_types::{
+    resolver::{StateStorageView, TModuleView, TResourceGroupView, TResourceView},
+    resource_group_adapter::{GroupSizeKind, ResourceGroupAdapter},
+};
+use bytes::Bytes;
+use move_core_types::{language_storage::StructTag, value::MoveTypeLayout};
+use std::collections::{BTreeMap, HashMap};
 
 /// Adapter to convert a `StateView` into an `ExecutorView`.
-pub struct ExecutorViewBase<'s, S>(&'s S);
+pub struct ExecutorViewBase<'s, S>(&'s S, ResourceGroupAdapter<'s>);
```

Comment:
if it has two fields, probably better to start naming them instead of using .0, .1

Reply:
ExecutorViewBase is gone, as per the new design :)